tests/aws/services/cloudwatch/test_cloudwatch.py

```diff
@@ -1391,7 +1391,7 @@ def contains_receive_delete_metrics() -> int:
 
     @markers.aws.validated
     @markers.snapshot.skip_snapshot_verify(
```

Comment:
Actually i think it would be better if we use a different approach here. Because the `DashboardArn` should not be skipped for the new provider.

Also adding a comment would be nice, so that we know why this change was necessary in the first place.
When we re-run snapshots, we sometimes test if the skips are still relevant, and remove them if the test works fine. The `DasboardEntries..Size` is one parameter where I think this could happen, so having a comment to make it clear why it is there would be great 🙏 

I would suggest 
```
        @markers.snapshot.skip_snapshot_verify(
        condition=is_old_provider,
        paths=[
            "$..DashboardArn",
        ],
        )
       @markers.snapshot.skip_snapshot_verify(
        paths=[
            "$..DashboardEntries..Size", # need to be skipped because size changes if the region name length is longer
        ]
    )
```

Reply:
Agreed. Thanks for the review @steffyP!